source/fundamentals/time-series.txt

```diff
@@ -20,8 +20,8 @@ Node.js driver.
 Create a Time Series Collection
 -------------------------------
 
-We recommend that you create a time series collection using the MongoDB Shell. 
-Refer to the :mdb-shell:`MongoDB Shell documentation<>` to learn more. 
+We recommend that you create a time series collection using the MongoDB Shell.
+Learn more about how to install and run the MongoDB Shell in the :mdb-shell:`MongoDB Shell documentation <>`.
 For detailed instructions on creating a time series collection
 using the MongoDB Shell, see our
 :manual:`MongoDB Manual entry on time series collections </core/timeseries-collections/#create-a-time-series-collection>`.
```

Comment:
I: this isn't technically part of the PR, but I think less text should be link text here

S: 
```suggestion
For detailed instructions on creating a time series collection
in the MongoDB Shell, see :manual:`Create and Query a Time Series Collection </core/timeseries/timeseries-procedures/>` in the Server manual.
```

Reply:
I've done back and forth on this and I'd like to stick with this text :)

Reply:
All good, I would still recommend changing the actual link though. there isnt a create-a-time-series-collection section on that page anymore, looks like it has been moved to /core/timeseries/timeseries-procedures/